### PR TITLE
diff page: status bar design refresh styles

### PR DESCRIPTION
**Pull request by 5h1rU**

### CODE REVIEW

File: client/web/src/extensions/components/StatusBar.tsx [hunk 1]
```diff
@@ -108,7 +108,12 @@ export const StatusBar: React.FunctionComponent<StatusBarProps> = ({
                         <LeftIcon className="icon-inline" />
                     </button>
                 )}
-                <div className="status-bar__items d-flex align-items-center px-2" ref={carouselReference}>
+                <div
+                    className={classNames('status-bar__items d-flex align-items-center', {
+                        'px-2': !isRedesignEnabled,
```
Comment (tjkandala) [inline client/web/src/extensions/components/StatusBar.tsx:113]: The `<Blob>`  status bar needs this padding. Should we add a `statusBarItemsClassName` prop to make status bar styling more extensible?

In this PR:
Screenshot from 2021-06-01 11-35-41

On main:
Screenshot from 2021-06-01 11-38-31

File: client/web/src/extensions/components/StatusBar.tsx [hunk 2]
```diff
@@ -83,7 +83,7 @@ export const StatusBar: React.FunctionComponent<StatusBarProps> = ({
     return (
         <div
             className={classNames(
-                'status-bar w-100 border-top d-flex',
+                'status-bar w-100 d-flex border-bottom',
```
Comment (tjkandala) [inline client/web/src/extensions/components/StatusBar.tsx:86]: I think we can get rid of the border utility class and apply it through `className`

**Review by tjkandala — COMMENTED** [review]
The diff status bar looks good! However, this approach can have unintended effects on other status bars, mainly the `<Blob>` status bar

**Comment by 5h1rU** [comment]
Those changes were addressed in a different PR.

**Pull request closed without merging**